app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_u = User.first
+  end
```

Comment:
- [ ] It would be better to add detailed instructions about how to run the app locally. Before running the server, we need to create, migrate before running the server, otherwise we will encounter errors

Reply:
@addod19 can we speak zoom

Reply:
Yeah sure!!! Send me a link invitation

Reply:
@addod19 thanks